crates/biome_service/src/file_handlers/json.rs

```diff
@@ -140,6 +140,9 @@ fn parse(
         allow_comments: parser.allow_comments
             || source_type.is_jsonc()
             || is_file_allowed_as_jsonc(rome_path),
+        allow_trailing_commas: parser.allow_trailing_commas
+            || source_type.is_jsonc()
+            || is_file_allowed_as_jsonc(rome_path),
```

Comment:
Would it be acceptable to enable trailing commas for `jsonc` by default?

Reply:
I checked original JSONC pasrer, the default value is false. I fixed it
ref: https://github.com/microsoft/node-jsonc-parser/blob/e82ebd90d91f026521d88d11c9749e2f02b9a79f/src/impl/parser.ts#L24